### DISCUSSION TITLE
How can we search multiple userId in the search filter of api/reg/search API in dockstaging?

### DISCUSSION THREAD

**Discussion by githubpiyush** [Sunbird-Lern/Community#109] [Query]
@reshmi-nair 
cc :- @aks30
**API** 
`https://dockstaging.sunbirded.org/api/reg/search`

**Body** 
`{
    "id": "open-saber.registry.search",
    "ver": "1.0",
    "ets": "11234",
    "params": {
        "did": "",
        "key": "",
        "msgid": ""
    },
    "request": {
        "entityType": [
            "User_Org"
        ],
        "filters": {
            "userId": {
                "eq": "d59873a0-40ea-461b-9402-ab090932f92d"
            }
        }
    }
}`

In the above body we are searching for only one userId.
Is there any way to search for multiple users in one search filter? 
If yes please share the filter.

**Comment by Jayaprakash8887** [comment]
Please raise this query in COKREAT BB forum: https://github.com/orgs/Sunbird-coKreat/discussions
